### PR TITLE
fix16384

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: larray/inout/excel.py [hunk 1]
```diff
@@ -320,20 +320,63 @@ def __setitem__(self, key, value):
...
-                left_cell = used.Cells(last_used_row, last_col + 1).End(xldir.xlToLeft)
-                if left_cell.Column > 1 or left_cell.Value is not None:
-                    break
-            for last_used_col in range(last_col, 0, -1):
-                up_cell = used.Cells(last_row + 1, last_used_col).End(xldir.xlUp)
-                if up_cell.Row > 1 or up_cell.Value is not None:
-                    break
-            return (last_used_row, last_used_col)
+
+            sheet = self.xw_sheet.api
```
Comment (alixdamman) [inline larray/inout/excel.py]: api = self.xw_sheet.api ?

File: doc/source/changes/version_0_26_1.rst.inc [hunk 2]
```diff
@@ -1,7 +1,9 @@
-Miscellaneous improvements
+﻿Miscellaneous improvements
 --------------------------
 
-* improved something.
+* Made handling sheets with many blank columns/rows after the data much faster (but still slower than sheets
```
Comment (alixdamman) [inline doc/source/changes/version_0_26_1.rst.inc]: s/Made handling sheets/Made handling Excel sheets/

File: larray/inout/excel.py [hunk 3]
```diff
@@ -280,15 +280,32 @@ def _fill_slice(s, length):
...
+        """
+        Parameters
+        ----------
+        key
+        obj : object
+            any object with a shape attribute
+
+        Returns
+        -------
+
```
Comment (alixdamman) [inline larray/inout/excel.py]: write something after Returns or do not add section Returns

File: larray/inout/excel.py (hunk 1 above)
Comment (gdementen) [reply larray/inout/excel.py]: sheet is more descriptive. It's a "raw" win32com sheet object.

File: larray/inout/excel.py (hunk 3 above)
Comment (gdementen) [reply larray/inout/excel.py]: ok will do that on Monday

File: doc/source/changes/version_0_26_1.rst.inc (hunk 2 above)
Comment (gdementen) [reply doc/source/changes/version_0_26_1.rst.inc]: ok

**Comment by gdementen** [comment]
improved docstring & changelog

**Comment by alixdamman** [comment]
LGTM

**Pull request merged**